README.md

```diff
@@ -11,89 +11,227 @@ Library used only for implementors of custom listeners for ReportPortal.
+```
+    
+
+
+## Example
 
-    npm install reportportal-client
+```javascript
+let RPClient = require('reportportal-client');
 
+let rpClient = new RPClient({
+    token: "00000000-0000-0000-0000-000000000000",
+    endpoint: "http://your-instance.com:8080/api/v1",
+    launch: "LAUNCH_NAME",
+    project: "PROJECT_NAME"
+});
+
+rpClient.checkConnect().then((responce) => {
+    console.log('You have successfully connected to the server.');
+    console.log(`You are using an account: ${responce.full_name}`);
+}, (error) => {
+    console.log('Error connection to server');
+    console.dir(error);
+});
+```
 
-## Usage
+## Settings
+When creating a client instance, you need to specify the following parameters:
+
+Parameter | Description
+--------- | -----------
+token     | user's token Report Portal from which you want to send requests. It can be found on the profile page of this user.
+endpoint  | URL of your server. For example, if you visit the page at 'https://server:8080/ui', then endpoint will be equal to 'https://server:8080/api/v1'.
+launch    | Name of lunch at creation.
+project   | The name of the project in which the lunches will be created.
+
+## Api
+Each method (except checkConnect) returns an object of a specific format:
+```javascript
+{
+    tempId: '4ds43fs', // generated by the client id for further work with the created item
+    promise: Promise // An object indicating the completion of an operation
+}
+```
+The client works synchronously, so to send the following requests it is not necessary to wait for the end of previous ones.
+
+### checkConnect
+ checkConnect - asynchronous method for verifying the correct connection of the client
+```javascript
+rpClient.checkConnect().then((responce) => {
+    console.log('You have successfully connected to the server.');
+    console.log(`You are using an account: ${responce.full_name}`);
+}, (error) => {
+    console.log('Error connection to server');
+    console.dir(error);
+});
+```
 
-Main classes are:
+### startLaunch
+startLaunch - starts a new launch, return temp id that you want to use for all items within this lunch.
+```javascript
+let launchObj = rpClient.startLaunch({
+    name: "Client test",
+    start_time: rpClient.helpers.now(),
+    description: "description of the launch",
+    tags: ["tag1", "tag2"]
+});
+console.log(launchObj.tempId);
+```
+The method takes one argument:
+* launch data object:
+
+Parameter | Description
+--------- | -----------
+start_time | (optional) start time launch(unix time). Default: rpClient.helpers.now()
+name      | (optional) lunch name. Default: parameter 'launch' specified when creating the client instance
+mode      | (optional) "DEFAULT" or "DEBUG". Default: "DEFAULT"
+description | (optional) description of the launch (supports markdown syntax)
+tags      | (optional) array of launch tags
+
+If you want to know the real id launch, wait for the method to finish (the real id does not participate in the client's work)
```

Comment:
To know the real launch id wait for the method to finish (the real id is not used by the client)